src/component/checkboxes.html

```diff
@@ -0,0 +1,16 @@
+<template>
+
+  <template containerless repeat.for="option of options">
+    <div class="checkbox">
+      <label>
+        <input
+        type="checkbox"
+        name.bind="option[property]"
+        model.bind="option.id"
```

Comment:
`model` bindable is used to populate the `value`. Whever a checkbox is checked this value is added to the array of items. It's also used to determine what checkboxes are initially checked.